src/scraper/Classes/APIGrabber.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+	namespace TopDog\scraper\Classes;
```

Comment:
weird indentation in this file, please sort out

Reply:
Jamila

Reply:
Done